### ISSUE TITLE
Enum name not read of x-ms-enum

### ISSUE THREAD

**Issue by pakrym** [Azure/autorest.modelerfour#320]
## header.json test server swagger
```
            "x-ms-enum": { "name": "GreyscaleColors", "modelAsString": false},
```

## Code model

![image](https://user-images.githubusercontent.com/1697911/87477507-1476dd80-c5dd-11ea-9c50-d381ffe9b9a6.png)

**Comment by daviwil** [comment]
This should be fixed now in `@autorest/modelerfour@4.15.396`, please give it a shot and let me know if it resolves the issue.  /cc @joheredi

**Comment by pakrym** [comment]
Fixed.
